src/app/add-child/add-child.component.html

```diff
@@ -0,0 +1,30 @@
+<form class="form-signin" [formGroup]="addForm" (ngSubmit)="onSubmit()">
```

Comment:
It would be much easier to create the form using BootStrap tags (and it would work well on mobile devices!). But I don't know any good tutorial to start with. So let's stick to more important topics. ;)